src/Runtime/Optimized/Physics/Bind/rigidBodyBundle.ts

```
@@ -159,12 +159,19 @@ export class RigidBodyBundle {
      */
-    public constructor(runtime: IPhysicsRuntime, info: RigidBodyConstructionInfoList) {
+    public constructor(runtime: IPhysicsRuntime, info: RigidBodyConstructionInfoList, countOverride?: number) {
         if (info.ptr === 0) {
             throw new Error("Cannot create rigid body bundle with null pointer");
         }
-        const count = info.count;
+        if (countOverride !== undefined) {
+            if (info.count < countOverride) {
+                throw new Error("Cannot create rigid body bundle with count greater than info count");
+            }
+            countOverride = Math.max(0, countOverride);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)